### ISSUE TITLE
`PRIMARY KEY NOT NULL AUTO_INCREMENT` parsed error

### ISSUE THREAD

**Issue by fannheyward** [taozhi8833998/node-sql-parser#709]
**Describe the bug**
`Expected "#", ")", ",", "--", "/*", "CHECK", "COLLATE", "COLUMN_FORMAT", "COMMENT", "CONSTRAINT", "REFERENCES", "STORAGE", or [ \t\n\r] but "N" found. [sql]`

**Database Engine**
mysql

**To Reproduce**

```mysql
create database cfb;

create table cfb.client (
    i_client INT PRIMARY KEY NOT NULL AUTO_INCREMENT,
    d_nas DATETIME
);
```

error on `NOT`

-the node-sql-parser version 3.6.3
-the node version 16.6.2

**Comment by taozhi8833998** [comment]
@fannheyward  I will fix it tomorrow.

**Comment by taozhi8833998** [comment]
@fannheyward I have published the `3.6.4` version to fix the issue, try it.
